arwen/host/execution.go

```diff
@@ -424,10 +424,10 @@ func (host *vmHost) callSCMethodIndirect() error {
 
 	_, err = function()
 	if err != nil {
-		return arwen.ErrFunctionRunError
+		err = host.handleBreakpointIfAny(err)
```

Comment:
I've always postponed doing this change, due to testing complexity. It is indeed required to handle breakpoints here, but there's probably 200 edge cases to watch out for. Adding a JIRA task to write tests for it...